### ISSUE TITLE
[Enhancement] New focus ring with three variants

### ISSUE THREAD

**Issue by Fuzzy3** [kirbydesign/designsystem#4427]
## Describe the enhancement
As part of making our components meet the WCAG requirement of contrast, we are going to introduce a new focus ring in three variants.

The three variants must replace the old solution.

## Describe the solution you'd like
Introduce three variants.

### 1) Focus Inner Shadow SM (inset)
For smaller controls like checkboxes, radios and toggles. 
<img width="326" height="45" alt="Image" src="https://github.com/user-attachments/assets/66243b07-39e0-4452-9f12-56db6c828b69" />
Style: `box-shadow: 0px 0px 0px 2px #238CEC inset, 0px 0px 0px 3px var(--System-background-color, #ffffff) inset;`

### 2) Focus Inner Shadow MD (inset)
For larger controls like buttons, segmented controls and dropdowns.
<img width="818" height="96" alt="Image" src="https://github.com/user-attachments/assets/729ac1a1-000f-4ec8-b238-043d7544ef82" />
Style: `box-shadow: 0px 0px 0px 2px #238CEC inset, 0px 0px 0px 4px var(--System-background-color, #ffffff) inset;`

### 3) Focus Outer Shadow MD 
For inline text links
<img width="358" height="57" alt="Image" src="https://github.com/user-attachments/assets/3833b619-a18d-40ce-ab25-270466c0ab1f" />
Style: `box-shadow: 0px 0px 0px 2px #238CEC inset, 0px 0px 0px 4px var(--System-background-color, #ffffff);`

### New Color
As part of this issue the focus ring color should also be updated to #238CEC.

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
